{{cookiecutter.project_name}}/project.yml

```diff
@@ -13,3 +13,4 @@ repo:
       url: {{cookiecutter.repo_url}}
 install_mode: {{cookiecutter.install_mode}}
 requires:
+pip: { use_config_file: False }
```

Comment:
This will break with the current stable version, won't it? If so I think we need to delay merging it until after the next release. Could you add this PR to the tickets for the next release epic?

Reply:
Indeed it will! I'll add it yes :)